src/core/services.ts

```diff
@@ -2,77 +2,75 @@ import { Sender } from 'xstate';
+            String.fromCharCode.apply(
+              null,
+              new Uint8Array(details.requestBody.raw[0].bytes) as any
             )
-          );
+          )
+        );
 
-          send({
-            type: 'ON_REQUEST',
-            payload: {
-              request: {
-                requestId: details.requestId,
-                query: queryDetails.query,
-                variables: queryDetails.variables
-              }
-            }
-          });
-        }
-      },
-      { urls: ['<all_urls>'] },
-      ['requestBody']
-    );
-
-    //@ts-ignore
-    chrome.webRequest.onCompleted.addListener(
-      function (details: any) {
         send({
-          type: 'ON_REQUEST_COMPLETE',
+          type: 'ON_REQUEST',
           payload: {
-            requestId: details.requestId,
-            statusCode: details.statusCode
+            request: {
+              requestId: details.requestId,
+              query: queryDetails.query,
+              variables: queryDetails.variables
+            }
           }
         });
-      },
-      { urls: ['<all_urls>'] }
-    );
+      }
+    },
+    { urls: ['<all_urls>'] },
+    ['requestBody']
+  );
 
-    //@ts-ignore
-    chrome.webRequest.onErrorOccurred.addListener(
-      function (details: any) {
-        send({
-          type: 'ON_REQUEST_ERROR',
-          payload: {
-            requestId: details.requestId,
-            statusCode: details.error === canceled_error ? 'canceled' : 'error'
-          }
-        });
-      },
-      { urls: ['<all_urls>'] }
-    );
+  //@ts-ignore
+  chrome.webRequest.onCompleted.addListener(
+    function (details: any) {
+      send({
+        type: 'ON_REQUEST_COMPLETE',
+        payload: {
+          requestId: details.requestId,
+          statusCode: details.statusCode
+        }
+      });
+    },
+    { urls: ['<all_urls>'] }
+  );
 
-    //@ts-ignore
-    chrome.webRequest.onBeforeSendHeaders.addListener(
-      function (details: any) {
-        send({
-          type: 'ON_BEFORE_SEND_HEADERS',
-          payload: {
-            requestId: details.requestId,
-            requestHeaders: details.requestHeaders
-          }
-        });
-      },
-      { urls: ['<all_urls>'] },
-      ['requestHeaders']
-    );
-  }
+  //@ts-ignore
+  chrome.webRequest.onErrorOccurred.addListener(
+    function (details: any) {
+      send({
+        type: 'ON_REQUEST_ERROR',
+        payload: {
+          requestId: details.requestId,
+          statusCode: details.error === canceled_error ? 'canceled' : 'error'
+        }
+      });
+    },
+    { urls: ['<all_urls>'] }
+  );
+
+  //@ts-ignore
+  chrome.webRequest.onBeforeSendHeaders.addListener(
+    function (details: any) {
+      send({
+        type: 'ON_BEFORE_SEND_HEADERS',
+        payload: {
+          requestId: details.requestId,
+          requestHeaders: details.requestHeaders
+        }
+      });
+    },
+    { urls: ['<all_urls>'] },
+    ['requestHeaders']
+  );
 };
```

Comment:
All this diff came from the fact we removed the surrounding `if` statement condition. Since we're not using Webpack Dev Server anymore, we're not testing it *outside* of an extension. So Chrome's WebRequest API is always available.